### PR TITLE
Don't batch a single call

### PR DESCRIPTION
**Pull request by bobo-k2**
**Pull Request Summary**
This PR contains the following:
- fix not to create a batch call for a single transaction
- background image for unstoppable grants category
- allow staking amount lower than min. staking amount if user already staked on the dApp

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

### CODE REVIEW

**Comment by Kahonnohak** [comment]
@bobo-k2 could we add unstoppable image on this PR?

File: src/staking-v3/logic/repositories/DappStakingRepository.ts [hunk 1]
```diff
@@ -397,7 +397,7 @@ export class DappStakingRepository implements IDappStakingRepository {
   public async batchAllCalls(calls: ExtrinsicPayload[]): Promise<ExtrinsicPayload> {
     const api = await this.api.getApi();
 
-    return api.tx.utility.batchAll(calls);
+    return calls.length == 1 ? calls[0] : api.tx.utility.batchAll(calls);
```
[the remark this answers is not in the document]
Comment (bobo-k2) [reply src/staking-v3/logic/repositories/DappStakingRepository.ts]: Thanks, fixed in https://github.com/AstarNetwork/astar-apps/pull/1184/commits/f0b10426697112e987920629421205dc5edc1799

**Review by gluneau — APPROVED** [review]
It works.

**Review by impelcrypto — APPROVED** [review]

**Review by Kahonnohak — APPROVED** [review]
LGTM

**Pull request merged**